### PR TITLE
Replace Laravel framework dependency with Illuminate packages

### PR DESCRIPTION
**Pull request by Szasza**
Resolving #7

### CODE REVIEW

**Comment by Szasza** [comment]
Hi @fntneves,

I didn't want to mess up the build jobs by modifying the tests, however it would be great if they could be modified so Lumen could be tested as well - Testbench is Laravel-only.

Cheers,
Szasza

**Comment by Szasza** [comment]
Hi @fntneves ,

Would it be possible to merge the PR if there are no comments from your end?

Thank you in advance.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea/
```
Comment (fntneves) [inline .gitignore]: This should be part of the global config, IMO.

File: composer.json [hunk 2]
```diff
@@ -4,13 +4,19 @@
...
+        "illuminate/support": "5.5.* || 5.6.*",
+        "php": "^7.0"
     },
     "authors": [
         {
             "name": "Francisco Neves",
             "email": "hi@francisconeves.me"
+        },
+        {
+            "name": "Szabolcs (Szasza) Palmer",
```
Comment (fntneves) [inline composer.json]: I thank you so much for your PR, however I see unpractical to add all contributors to this file.

I will add credits to the README, pointing to contributors.

File: .gitignore (hunk 1 above)
Comment (Szasza) [reply .gitignore]: Done.

File: composer.json (hunk 2 above)
Comment (Szasza) [reply composer.json]: Removed.

**Comment by Szasza** [comment]
Hi @fntneves ,

My pleasure.

Thank you for the response, I have done the required changes.

Could you please re-review?

**Comment by Szasza** [comment]
Hi @fntneves ,

Great, thank you for the merge, looking forward to the new release :+1:

**Comment by fntneves** [comment]
FIY, I'm working on this. I found a bug and I am fixing it righ now. It will take some extra time.

Hopefully, it will be released on this weekend.

**Comment by fntneves** [comment]
Hi @Szasza,

This is now released. 
There was no bug, however, it was a false alarm caused by a misconfiguration in Lumen.

Thank you again for your contribution!

**Comment by Szasza** [comment]
Hi @fntneves ,

Awesome, my pleasure, and thank you for the new release!

**Pull request merged**